pulp_smash/tests/docker/cli/utils.py

```diff
@@ -0,0 +1,118 @@
+# coding=utf-8
+"""Common utilities that assist in testing the CLI with docker repositories."""
+from __future__ import unicode_literals
+
+import unittest2
+
+from pulp_smash import cli, config, utils
+
+
+FEED = 'https://index.docker.io'
```

Comment:
Ah I didn't realize there were constants. Those values are no longer correct, and also there is now one endpoint that does v1 and v2. I'll change my PR to update those constants. We might as well keep them separate in case docker changes them again later.